src/content/blam/tags/readme.md

```diff
@@ -1,7 +1,18 @@
 ---
 title: Tags
+template: tagIndex
```

Comment:
Using the new `tagIndex` page template for rendering tables of tags _after_ this markdown content: